route/TAS.py

```diff
@@ -154,3 +155,23 @@ def init_TAS(self):
         self.sequencer = SequencerEngine(
             window=self.window, config=self.config_data, root=TAS_root
         )
+
+
+class SoSBattleTestMenu(TASMenu):
```

Comment:
I suggest overriding the `custom_gui()` method on this one and add some text in the GUI as to what location the function is supposed to be run.